test/e2e/cli_test.go

```diff
@@ -530,33 +568,23 @@ func (s *CLISuite) TestWorkflowLint() {
 		err = ioutil.WriteFile(filepath.Join(tmp, "my-workflow.yaml"), data, 0644)
 		s.CheckError(err)
 		s.Given().
-			RunCli([]string{"lint", tmp}, func(t *testing.T, output string, err error) {
+			RunCli("", []string{"lint", tmp}, func(t *testing.T, output string, err error) {
 				if assert.NoError(t, err) {
 					assert.Contains(t, output, "my-workflow.yaml is valid")
 				}
 			})
 	})
-
-	s.Run("Different Kind", func() {
```

Comment:
the removal of this test shows one change in behaviour - that we would silently ignore this now